src/conduits/install-rsc-conduit.ts

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs-extra';
+import {DlpOpenConduitReqType} from '../protocols/dlp-commands';
+import {DlpConnection} from '../protocols/sync-connections';
+import {DATABASES_STORAGE_DIR, TO_INSTALL_DIR} from '../sync-utils/sync-device';
+import {ConduitData, ConduitInterface} from './conduit-interface';
+import {writeDbFromFile} from '../sync-utils/write-db';
+import debug from 'debug';
+
+const log = debug('palm-sync').extend('conduit').extend('install-rsc');
+
+/**
+ * This conduit installs every resource that is present in the Palm's
+ * install dir
+ */
+export class InstallNewResourcesConduit implements ConduitInterface {
+  getName(): String {
+    return 'install new resources from PC';
+  }
+  async execute(
+    dlpConnection: DlpConnection,
+    conduitData: ConduitData
+  ): Promise<void> {
+    await dlpConnection.execute(DlpOpenConduitReqType.with({}));
+    let toInstallDir = fs.opendirSync(
+      `${conduitData.palmDir}/${TO_INSTALL_DIR}`
```

Comment:
In general it's best practice to use `path.join()` to construct paths instead of string operations, here and everywhere else :smile: 

If too much work I can also fix this after we merge the PR.

Reply:
Damn, TIL! That's so much more convenient that what I did haha Fixed!